.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Great job adding test.md to your `.gitignore` file, however I kindly suggest that you add `node_modules/` also to your .gitignore file as written in the project requirements.